charts/calico/values.yaml

```diff
@@ -2,25 +2,34 @@
 version: master
 
 # Configure the images to use when generating manifests.
+# If a new image is added, update manifests/generate.sh
 node:
-  image: quay.io/calico/node
+  image: node
+  registry: quay.io/calico
```

Comment:
I think that everywhere else, we consider the registry to be `quay.io` and the image to be `calico/node` - we should be consistent here as well.

Reply:
Actually, monorepo considers registry to be `quay.io/calico`. See [defaultRegistries](https://github.com/projectcalico/calico/blob/master/release/pkg/manager/calico/manager.go#L42-L48)

There is discrepancy with operator which does use `quay.io` as registry and `tigera/operator` as image but aligning them is beyond the scope of this PR.

Reply:
This is what our other values.yaml file looks like: https://github.com/projectcalico/calico/blob/master/charts/tigera-operator/values.yaml#L81-L84

```
tigeraOperator:
  image: tigera/operator
  version: master
  registry: quay.io
```

It looks like the `DEV_REGISTRIES` env var and the `release/` tooling use the word registry to describe `<REGISTRY>/<REPOSITORY>` but that's just incorrect.

We don't need to fix those in this PR, but we shouldn't propagate the incorrect version, especially because there is another counter example in this repository that does it the "right" way.

Reply:
calicoctl is also incorrect in the tigera-operator values.yaml, so it's clearly all just a mess:

https://github.com/projectcalico/calico/blob/a31851348d5d65b573a6165354c5eb5cdd9c9704/charts/tigera-operator/values.yaml#L85-L87

```
calicoctl:
  image: quay.io/calico/ctl
  tag: master
```

Reply:
So, fine to do as you have because it's all a mess. Just makes me very sad.